### PR TITLE
DLS-7473 Make unexpected exceptions visible in Kibana

### PR DESCRIPTION
**Pull request by LAJW**
See: https://github.com/hmrc/individuals-income-api/pull/207

### CODE REVIEW

File: app/uk/gov/hmrc/individualsmatchingapi/controllers/CommonController.scala [hunk 1]
```diff
@@ -42,6 +43,8 @@ import scala.util.{Failure, Success, Try}
 
 abstract class CommonController @Inject()(cc: ControllerComponents) extends BackendController(cc) {
 
+  val logger: Logger = Logger(getClass)
```
Comment (johno1985) [inline app/uk/gov/hmrc/individualsmatchingapi/controllers/CommonController.scala:46]: can we use the Logging mixin instead? `with Logging`

**Review by johno1985 — COMMENTED** [review]
comment added

File: app/uk/gov/hmrc/individualsmatchingapi/controllers/CommonController.scala (hunk 1 above)
Comment (lutnos) [reply app/uk/gov/hmrc/individualsmatchingapi/controllers/CommonController.scala:46]: We should indeed, getClass is for pre- Play-2.8, or was it pre- Play-2.7?

**Comment by johno1985** [comment]
needs conflict resolution

**Review by johno1985 — APPROVED** [review]

**Pull request merged**